charts/nidhogg/chart/values.yaml

```
@@ -2,6 +2,10 @@ yggdrasil:
   repoURL: "https://github.com/distributed-technologies/yggdrasil.git"
   targetRevision: "main"
 
+cni:
+  serviceMonitor:
+    enabled: false
+
 installCNI: true
```

Could we changes this now that we're making changes? 

```suggestion

cni:
  enabled: true
  serviceMonitor:
    enabled: false
```